lib/Basic/CmdLineOptions.cpp

```diff
@@ -70,6 +70,15 @@ cl::opt<bool>
 CoreSolverOptimizeDivides("solver-optimize-divides", 
                           cl::desc("Optimize constant divides into add/shift/multiplies before passing to core SMT solver (default=off)"),
                           cl::init(false));
+llvm::cl::opt<bool>
+    UseMerge("use-merge",
```

Comment:
`CmdLineOptions` is for shared KLEE/Kleaver options, and thus is used for solver-related command-line options.
I guess the best place where to put these declarations is `MergeHandler`.

Reply:
This is a good catch, thanks, @andreamattavelli .  
@futile , @corrodedHash , if you can fix this quickly, that would be great.  Otherwise, I'd still prefer to merge this PR sometime tomorrow and we can fix this and other smaller issues separately.

Reply:
Good point! We can fix this by Monday (or Tuesday at the latest), would that be fast enough?

Reply:
That would work well, we can wait until Monday or Tuesday, thanks!